### ISSUE TITLE
SelectOneButton: unable to select elements with NVDA turn on

### ISSUE THREAD

**Issue by ntloc-axonivy** [primefaces/primefaces#13989]
### Describe the bug

![Image](https://github.com/user-attachments/assets/c606ff2b-7965-4dec-9fc5-563c0f1083f8)

When I turn on NVDA, I’m unable to select elements using Space or Enter. While without NVDA, it works fine.
Do we have any suitable solutions for this issue?
Thank you!

### PrimeFaces edition

None

### PrimeFaces version

v15.0.6-SNAPSHOT

### JSF implementation

Mojarra

### JSF version

2.2.15

### Java version

17.0.14

**Comment by melloware** [comment]
@ntloc-axonivy this feels like an NVDA bug I would report to NVDA https://github.com/nvaccess/nvda/issues

you can see they have over 2100 open issues.

**Comment by melloware** [comment]
Just looking through their issues this looks like your issue: https://github.com/nvaccess/nvda/issues/15802

**Comment by ntloc-axonivy** [comment]
@melloware thank you for your link. 

Because NVDA will switch to focus mode when a role is present so I found a workaround solution:

```javascript
document.querySelectorAll('.ui-selectonebutton .ui-button').forEach((btn) => {
    btn.setAttribute('role', 'radio');
});
``` 

I think PrimeFaces should enhance this component.

**Comment by melloware** [comment]
PR submitted will be fixed in 15.0.6
